Python/pywarpx/picmi.py

```diff
@@ -23,6 +23,7 @@
 # dictionary to map field boundary conditions from picmistandard to WarpX
 BC_map = {
     'open':'pml', 'dirichlet':'pec', 'periodic':'periodic', 'damped':'damped',
+    'silver_mueller':'absorbing_silver_mueller',
```

Comment:
Let me know if we shall keep the long name:
```suggestion
    'absorbing_silver_mueller':'absorbing_silver_mueller',
```

Reply:
I would use the long name to be consistent with the name used in input files, since (at least for now) this is only used in WarpX (and not yet supported in standard PICMI).

Reply:
But I want to add it to PICMI :)
See https://github.com/picmi-standard/picmi/pull/90

Otherwise it would not be well documented.

Reply:
Updated for longer name.